### PR TITLE
ZF1, ZF2: call parent::_after in _after to cleanup after each test

### PR DESCRIPTION
**Pull request by Naktibalda**
I had an issue with client carrying over domain from test which uses domain to test which doesn't.

### CODE REVIEW

File: src/Codeception/Module/ZF1.php [hunk 1]
```diff
@@ -147,6 +148,8 @@ public function _after(TestCase $test)
         \Zend_Session::$_unitTestEnabled = true;
         $this->queries = 0;
         $this->time = 0;
+
+        parent::_after($test);
```
Comment (DavertMik) [inline src/Codeception/Module/ZF1.php]: but why do you call it in `before`?

Comment (Naktibalda) [inline src/Codeception/Module/ZF1.php]: Miscommunication between Github and you.
_after starts in line 134

Comment (DavertMik) [inline src/Codeception/Module/ZF1.php]: yeah. right. That was how github showed it.

**Comment by DavertMik** [comment]
yeah. sorry. Please merge then

**Pull request merged**